.tsan-suppressions

```diff
@@ -1 +1,2 @@
-race:mongoc_counter*
\ No newline at end of file
+race:mongoc_counter*
+race:_mongoc_handshake_freeze
```

Comment:
Suppress the pre-existing race condition which is detailed in this ticket:
https://jira.mongodb.org/browse/CDRIVER-3685